draft-irtf-cfrg-hpke.md

```diff
@@ -1138,16 +1196,22 @@ the PSK or AuthPSK mode, is proven in {{HPKEAnalysis}}; in a quantum
 setting, the remaining security level is smaller and defined by the
 post-quantum security level of the AEAD scheme.
 
-## Security Requirements on a KEM used within HPKE
+## Security Requirements on a KEM used within HPKE {#kem-security}
+
+A KEM used within HPKE MUST allow HPKE to satisfy its desired security
+properties described in {{sec-properties}}. There are indications that
+this is the case
```

Comment:
"There are indications" is quite vague.  What indications?

Or do you mean "the following criteria are sufficient to meet HPKE's security needs"?

Reply:
Yes, it basically means "the following criteria are sufficient to meet HPKE's security needs". However, there is no proof yet that these criteria are actually sufficient for HPKE. Hence “indications”.

In Section “Security Considerations” we already have the formulation “The results from {{CS01}} provide some indication that any IND-CCA2-secure KEM will suffice here, but are not conclusive given the differences in the schemes.”; that is from where I borrowed “there are indications”.

These indications come from `CS01` and `SigncryptionDZ10`, as listed in the list elements.

Would it be better already if we said “{{CS01}} and {{SigncryptionDZ10}} provide some indications that the following criteria are sufficient to meet HPKE's security needs”?